test.dot

```diff
@@ -0,0 +1,51 @@
+digraph G {
```

Comment:
I'm trying to understand a bit what test.dot represents. Is this attempting to split the information out? Or are you just testing a different structure for constructing the overall graph using graphviz instead of plantuml?

Reply:
Just testing, should not be on the merge.